neoload/commands/test_results.py

```diff
@@ -101,6 +101,32 @@ def junit(__id, junit_file):
     displayer.print_result_junit(json_result, json_sla_test, json_sla_interval, junit_file)
 
 
+def get_sla_data_by_name_or_id(name):
+    if name == "cur":
+        name = user_data.get_meta(meta_key)
+    is_id = tools.is_id(name)
+
+    __id = tools.get_id(name, __resolver, is_id)
```

Comment:
This code is called every 5 seconds by the command fastfail, and the ID will be retrieved each time... This is not optimized because get_id will eventually call the API if "name" is not an ID. We can avoid this call by doing the resolution directly in the fastfail command before the monitor_loop.

Reply:
Resolved with 843a026 and 184006c